Vizzuality/marxan-cloud
# feat(projects): planning area from grid

## Substitute this line for a meaningful title for your changes

### Overview

**BACKEND:**

- [x] Upload custom planning area shapefile with grid.

**FRONTEND:**

- [x] Dropdown to select the way of setting planning area when creates a project.
- [ ] Custom planning area with grid Uploader.

### Designs

[Figma Designs](https://www.figma.com/file/hq0BZNB9fzyFSbEUgQIHdK/Marxan-Visual_V02?node-id=6302%3A11325)
[Figma Interaction Demo](https://www.figma.com/proto/hq0BZNB9fzyFSbEUgQIHdK/Marxan-Visual_V02?page-id=2991%3A66&node-id=6302%3A11601&viewport=243%2C48%2C0.25&scaling=min-zoom&starting-point-node-id=2991%3A68&show-proto-sidebar=1)

### Testing instructions

- You must create a project from the beginning and add the planning area as a grid. 
- Make sure later that the added planning area can be used correctly when creating a scenario within that project.

### Feature relevant tickets

**FE:** 
[MARXAN-823](https://vizzuality.atlassian.net/browse/MARXAN-823)

---
